pkg/distributor/http.go

```diff
@@ -73,7 +73,7 @@ func (d *Distributor) pushHandler(w http.ResponseWriter, r *http.Request, pushRe
 				"msg", "push request successful",
 			)
 		}
-		w.WriteHeader(http.StatusNoContent)
+		w.WriteHeader(http.StatusOK)
```

Comment:
This would change the response status code for both the `/otlp/v1/logs` and the regular `/api/v1/push` endpoints. We definitely don't want to change the latter.

You would need to provide the return status code as argument in the `pushHandler` function.